docs/Design System Generation.md

```diff
@@ -0,0 +1,52 @@
+## Design System
+
+The design system contains dimensions, colors and color modes in Figma. Because the amount of these variables makes it 
+hard to reflect the changes manually it is a better use of time to automate it.
+
+## AI Automation
+
+The idea is to output the variables into a json and then have AI (i.e. chatgpt) generate kotlin code from it.
+
+## Figma variables into json
+
+Use 'Variables Exporter For Dev Mode' figma plugin. It outputs all variables into a simple structured json.
```

Comment:
Can we add a reference to the tool's homepage?

Reply:
done